src/ape/options.py

```diff
@@ -1,6 +1,44 @@
 import click
+import click_logging
 
 from ape import networks
+from ape.logging import logger as logger
+from ape.utils import Abort
+
+
+class PluginHelper:
```

Comment:
I was even wondering about making the managers that you normally access via `from ape import <manager<` accessible here, but maybe that is not that useful.

Reply:
See about, thinking of more descriptive names for this :thinking: